Packages/ohif-studies/imports/client/lib/retrieveStudyMetadata.js

```diff
@@ -6,6 +6,19 @@ import 'meteor/ohif:viewerbase';
 // promises and prevent unnecessary subsequent calls to the server
 const StudyMetaDataPromises = new Map();
 
+
+/**
+ * Delete the study promise cached to ensure next retrieve will get it retrieved by server call
```

Comment:
This isn't super clear to me. Could you change it to:

'Delete the cached study metadata retrieval promise to ensure that the browser will re-retrieve the study metadata when it is next requested.'